### PR TITLE
add Lasagne Bolognaise

### PR DESCRIPTION
**Pull request by rsimai**

### CODE REVIEW

File: recipes.adoc [hunk 1]
```diff
@@ -26,12 +26,27 @@ ifndef::env-github[]
...
 
 Side dishes complete a meal, providing a compliment to the entree like
 the bass line in a song... essential components for a full experience.
 
+ifdef::env-github[]
+<<recipes/sides/raspberry_pretzel_salad.adoc#, Raspberry Pretzel Salad>>
+endif::[]
+ifndef::env-github[]
+include::recipes/sides/raspberry_pretzel_salad.adoc[]
+endif::[]
```
Comment (jeremy-moffitt) [inline recipes.adoc]: I'm confused about this block being here, perhaps this change needs rebased?

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
I think deleting lentil_dal.adoc is an artifact of some attempted refactoring? Also appears to have some of the changes for the raspberry pretzel salad

File: recipes.adoc [hunk 2]
```diff
@@ -26,12 +26,27 @@ ifndef::env-github[]
...
+
```
Comment (aosthof) [inline recipes.adoc]: The raspberry section should not be part of this PR.

File: recipes.adoc (hunk 1 above)
Comment (rsimai) [reply recipes.adoc]: yeah, Alex just asked me the same and I have no idea ...

**Review by aosthof — COMMENTED** [review]
You also might want to rotate IMG_0831.jpg and IMG_0831.jpg.

**Review by aosthof — APPROVED** [review]
Being picky - you mix upper and lower case in the list of ingredients (and in some other places). Aside from that, LGTM. Typos can also be fixed at a later point in time and should not prevent this PR from being merged.

**Review by jeremy-moffitt — APPROVED** [review]
looks ok to me!

**Comment by jeremy-moffitt** [comment]
just realized main dishes was added (in a PR I approved of all things). I think once things settle I'll move "entrees" to "main"
